cdk/stack.py

```diff
@@ -52,6 +53,9 @@ def __init__(
             "CLIENT_ID": config.client_id,
             "CLIENT_SECRET": config.client_secret,
             "MWAA_ENV": config.airflow_env,
+            "RASTER_URL": config.raster_url,
+            "OIDC_PROVIDER_ARN": config.oidc_provider_arn,
+            "OIDC_PROVIDER_REPO_ID": config.oidc_repo_id,
```

Comment:
can we be mindful about what env we're sending where instead of having a catchall env for all? there are two lambdas in this stack that are getting all these env variables but they don't need all these env vars, especially the OIDC vars